### ISSUE TITLE
[Bug] Pinned items are not available on smaller screens

### ISSUE THREAD

**Issue by aledeg** [shaarli/Shaarli#1347]
The pin action disappears when shrinking the size of the display. What I can see in the DOM is that there is two action toolbars that are displayed alternatively depending on the size of the display. The one for smaller screens does not have the pin action. Thus it feels like it disappears.

I am currently using 0.11.0
